backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/TestResult.java

```diff
@@ -1,5 +1,5 @@
 package gov.cdc.usds.simplereport.db.model.auxiliary;
 
 public enum TestResult {
-	POSITIVE, NEGATIVE, UNDETERMINED;
-}
\ No newline at end of file
+	POSITIVE, NEGATIVE, UNDETERMINED
```

Comment:
Please don't do things like this, it blows up the diff for no reason.

Reply:
Agreed. My IDE complained that it was redundant, but I'll put it back.